CMakePresets.json

```diff
@@ -0,0 +1,133 @@
+﻿{

```

Comment:
Using the CMakePresets.json requires CMake 3.20 or later. On WSL, you also need to have the ``ninja-build`` installed.

IF you don't have these, you can still use older cmake versions the original 'manual 'way.